portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,74 @@
+*{
+    margin: 0;
+    box-sizing: border-box;
+}
+body{
+    padding: 0;
+    margin: 0;
+    position: relative;
+    background: linear-gradient(to right, #4b6cb7, #182848);
+}
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+#greeting-container {
+    margin-top: 20px;
+}
+#navbar{
+    display: inline-block;
+    width: 100%;
+    background: black;
+    display: flex;
+    flex-direction: row;
+    padding: 1rem;
+    color: rgb(95, 97, 226);
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 1.5rem;
+    font-weight: 800;
+    text-transform: uppercase;
+}
+img{
+    height: auto;
+    min-height: 300px;
+    width: 50vw;
+    min-width: 300px;
+    border: solid black 2px;
+    border: solid rgb(0, 255, 76) 6px;
+}
+h1{
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    font-weight: 600;
+    margin: 2rem 0 1rem 0;
+    background: linear-gradient(to right, #1cd8d2, #33ec9f);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+p{
+    font-family:'Lucida Sans', 'Lucida Sans Regular',
+    'Lucida Grande', 'Lucida Sans Unicode',
+    Geneva, Verdana, sans-serif;
+    text-align: center;
+    margin: 10px 0;
+    color: white;
+}
+a{
+    color: red;
 }
 
-#greeting-container {
-  margin-top: 20px;
+button{
+    padding: 4px 8px;
+    margin: 4px;
+    border: solid rgb(244, 248, 14) 4px;
+    outline: none;
+    border-radius: 10px;
+    background: yellow;
 }
+button:hover{
+    cursor: pointer;
+}
```

Comment:
Couple formatting violations:
- "Indent by two spaces at a time." (might wanna configure your editor to use 2 space indentation) - https://google.github.io/styleguide/htmlcssguide.html#Indentation
- "Use a space between the last selector and the declaration block." - https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation
- "Separate rules by new lines." - https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation